iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -539,7 +539,8 @@ - (void)messageViewDidSelectAction:(OSInAppMessage *)message withAction:(OSInApp
     if (action.clickUrl)
         [self handleMessageActionWithURL:action];
     
-    [self handlePromptActions:action.promptActions];
+    if (action.promptActions.count > 0)
```

Comment:
Should we nil check just in case?
`if (action.promptActions && action.promptActions.count > 0)`